source/class/qx/ui/form/FileSelectorButton.js

```diff
@@ -157,7 +157,10 @@ qx.Class.define("qx.ui.form.FileSelectorButton", {
       );
       input.addListenerOnce("appear", e => {
         let inputEl = input.getDomElement();
-        input.setStyle('display', 'none');
+       // since qx.html.Node does not even create the
+       // domNode if it is not set to visible initially
+       // we have to quickly hide it after creation.
+       input.setVisible(false);
```

Comment:
fix indentation

```suggestion
        // since qx.html.Node does not even create the
        // domNode if it is not set to visible initially
        // we have to quickly hide it after creation.
        input.setVisible(false);
```